pkg/didcomm/protocol/didexchange/persistence.go

```diff
@@ -103,14 +113,65 @@ func (c *ConnectionRecorder) GetConnection(connectionID string) (*ConnectionReco
 	return &ConnectionRecord{State: string(name)}, nil
 }
 
-// invitationKey computes key for invitation object
-func invitationKey(verKey string) (string, error) {
-	storeKey, err := computeHash([]byte(verKey))
+// saveConnectionRecord saved the connection record against the connection id  in the store
+func (c *ConnectionRecorder) saveConnectionRecord(invitation *Invitation,
+	msg *message) error {
+	connRecord := &ConnectionRecord{ThreadID: msg.ThreadID,
```

Comment:
I feel connection record should be passed into the function since this is a persistence layer. It should be up-to the business layer to populate the data. This is something similar to my [previous comment](https://github.com/hyperledger/aries-framework-go/pull/443#discussion_r332667202).

Reply:
done

Reply:
Is the patch pushed ? I don't see any new patchset.

Reply:
its pushing give me a minute sorry

Reply:
pushed now